File: trabajos en GO/ley_ohm.go
package main

import "fmt"

func main() {
	var v, r, i float64
	fmt.Print("Ingrese el valor de la tension (v): ")
	fmt.Scan(&v)
	fmt.Print("Ingrese el valor de la resistencia (r): ")
	fmt.Scan(&r)
	i = v / r
	fmt.Println("al conectar una resistencia de", r, "Ohm con una tension de", v, "V, la intensidad de la corriente es de", i)
}

File: trabajos en rubi/ley_ohm.rb
print "Ingrese el valor del voltaje del circuito: "
V = gets.chomp.to_i
puts
print "Ingrese el valor de la resistencia del circuito: "
R = gets.chomp.to_i
puts
I = V/R
puts "al conectar un voltaje de #{V} volts y una resistencia de #{R} ohms, la intensidad del circuito es de #{I} amperios"

print ("fin de la ejecucion_ escriba exit para salir:")
Z = gets.chomp.to_i
